src/sprints/sprints.service.ts

```diff
@@ -71,9 +71,12 @@ export class SprintsService {
                 },
             },
         });
-        return sprintsByTeamId?.voyage?.sprints?.filter(
+
+        const sprintId = sprintsByTeamId?.voyage?.sprints?.filter(
             (s) => s.number === sprintNumber,
         )[0]?.id;
+
+        return sprintId;
```

Comment:
We can return only ` null ` when  `sprintId` is not found rather than having both types `null` and `undefined`

```
   const sprintId = sprintsByTeamId?.voyage?.sprints?.filter(
            (s) => s.number === sprintNumber,
        )[0]?.id || null;
```

Reply:
I'd rather keep it undefined if it's undefined, null and undefined are different.

Although it looks like we can remove null

Reply:
Yeah  we can remove either of one .